tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -193,16 +196,24 @@ def __init__(self,
             # Single-producer tests are slower, bottlenecked on the
             # client side.
             self.expect_single_bandwidth = 200E6
+
+            if tiered_storage_enabled:
+                # We read very tiny segments 32KiB over high latency link
+                # (10ms to 100ms) with a concurrency limit of cloud_storage_max_connections=20`.
+                # Using Little's Law we can derive the arrival rate as `20/0.1`
+                # which is to say 200 segments per second. 200 * 32KiB = 6.25MiB
+                # per node. This is if we ignore all other sources of latency,
+                # contention, and S3 rate limiting or instabilities.
+                self.expect_bandwidth = node_count * 6 * 1E6
```

Comment:
This seems fairly low. Shouldn't we just double or triple `cloud_storage_max_connections` then? 20 connections seems extremely low.

Also is latency to S3 really 10 to 100ms? That seems quite high.

Reply:
We don't have ready metrics for this but anecdotally that's the ballpark.